### PR TITLE
Project structure changes and build script

### PR DESCRIPTION
**Pull request by mkucherenko**

### CODE REVIEW

**Comment by yuhanniu** [comment]
when I run "build.sh" I get:
```
EEEEEEEEEEEEEEEEEEEEEEEEPHP Fatal error:  Class 'Io\Token\Proto\Common\Member\CreateMemberType' not found in /Users/yh/token-repo/sdk-php/lib/TokenClient.php on line 105
```

**Comment by yuhanniu** [comment]
Some more regarding namespace and structure:
* Move `TestUtil` class to the tests folder, i.e. its namespace should be `Test/Tokenio`
* Move `PagedList` class out of the `Util` folder, its namespace should be `Tokenio`
* Remove the `Rpc/Request` namespace, move all the classes to the Tokenio` namespace.

**Comment by yuhanniu** [comment]
Also after I run the `build.sh` script, a folder called `Utiltests` is created under the `lib` folder. If it is created for testing purpose it should be under the `tests` folder.

**Comment by mkucherenko** [comment]
> Some more regarding namespace and structure:
> 
> * Move `TestUtil` class to the tests folder, i.e. its namespace should be `Test/Tokenio`
> * Move `PagedList` class out of the `Util` folder, its namespace should be `Tokenio`
> * Remove the `Rpc/Request` namespace, move all the classes to the `Tokenio` namespace.
> * Since we are making a bunch of breaking changes anyway, let's just remove the `TokenIO` and `TokenIOBuilder` class.

Done

**Comment by mkucherenko** [comment]
> Also after I run the `build.sh` script, a folder called `Utiltests` is created under the `lib` folder. If it is created for testing purpose it should be under the `tests` folder.

Added removing temp keys folders in tests `tearDown()`

File: build.sh [hunk 1]
```diff
@@ -8,14 +8,15 @@ rm -rf ./vendor
 composer install
 composer dump-autoload
 
+#Download doc generator tool
+curl -o ./tools/sami.phar http://get.sensiolabs.org/sami.phar
```
Comment (yuhanniu) [inline build.sh]: let's just add this to the repo

**Pull request merged**